### PR TITLE
[Gradle 7.0 migration] Upgrade gradle wrapper

### PR DESCRIPTION
**Pull request by tagantroy**
Use latest Gradle version.
This PR depends on #483

### CODE REVIEW

**Comment by Malinskiy** [comment]
Should probably contain updates to samples as well?

**Comment by tagantroy** [comment]
> Should probably contain updates to samples as well?

I'll update  sample apps in separate diff. It will require some extra work on gradle plugin.

File: buildSrc/src/main/kotlin/Versions.kt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 object Versions {
     val marathon = System.getenv("DEPLOY_VERSION_OVERRIDE") ?: "0.7.0"
 
-    val kotlin = "1.4.20"
+    val kotlin = "1.4.31"
```
Comment (Vacxe) [inline buildSrc/src/main/kotlin/Versions.kt:4]: 1.4.32 - latest stable

**Review by Vacxe — APPROVED** [review]

File: buildSrc/src/main/kotlin/Versions.kt (hunk 1 above)
Comment (tagantroy) [reply buildSrc/src/main/kotlin/Versions.kt:4]: I checked release notes of Gradle 7.0, they upgraded Kotlin to 1.4.31(
[#16330] - Upgrade embedded Kotlin to 1.4.31). I hope upgrade to 1.4.32 will not cause any issue, but I would prefer to keep version same. Let's check it later

**Pull request merged**